LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2016-2018 Damian Stasik <github@coded.pl>
+Copyright (c) 2022 Gamma Communications
```

Comment:
Please use the full company name "Gamma Communications GmbH" here as required by law. Also, you can use the e-mail address webdevelopment@gammacommunications.de here.